### ISSUE TITLE
[BUG] All tailwind classes are marked as out-of-order

### ISSUE THREAD

**Issue by qortex** [francoismassart/eslint-plugin-tailwindcss#118]
**Describe the bug**
Even after linting & fixing all tailwind class order, they are all linted as "warning" by vscode eslint language service.

**Expected behavior**
No lint error should be shown.

**Screenshots**

![image](https://user-images.githubusercontent.com/37897072/152600327-df0d8754-b364-4747-9823-803d003f83c2.png)

```bash
$ npx prettier --check libs/ui-kit/form-blocks/src/lib/components/input-checkbox/input-checkbox.component.html 
Checking formatting...
All matched files use Prettier code style!
```

**Environment (please complete the following information):**
 - OS: Linux Fedora
 - Softwares + version used:
     -  vscode 1.63.2
     -  Node 14.17.0
     - Nx 13.7.1 

**Additional context**
Nx & Angular.
Also using `"extends": ["prettier"],` in my eslint config.

I understand from this issue that the root cause might be identified? https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/31
Posting it here though, in case it needs to be tracked (or I'm mistaken).

**Comment by qortex** [comment]
Actually no, removing https://github.com/tailwindlabs/prettier-plugin-tailwindcss does not interfere, I have mixed up things in my head.

**Comment by francoismassart** [comment]
Hi @qortex,
If you want to use both `eslint-plugin-tailwindcss` and `prettier-plugin-tailwindcss` you must disable "tailwindcss/classnames-order" rule because both plugin use a different ordering system.

in your `.eslintrc` file:

```js
    "tailwindcss/classnames-order": "off",
```

**Comment by qortex** [comment]
Aw ok makes sense, thanks.
(Tried that actually but put it in the `*.ts` rules so it didn't have any effect -_- )
